HackIllinois/Models/CoreDataModels/Event+CoreDataClass.swift

```diff
@@ -18,12 +18,11 @@ public class Event: NSManagedObject {
     convenience init(context moc: NSManagedObjectContext, event: HIAPIEvent, locations: NSSet) {
         guard let entity = NSEntityDescription.entity(forEntityName: "Event", in: moc) else { fatalError() }
         self.init(entity: entity, insertInto: moc)
-        favorite = event.favorite
-        end = event.end
-        id = event.id
-        info = event.info
-        name = event.name
-        start = event.start
+        self.favorite = event.favorite
```

Comment:
Considered bad style to add self when its unnecessary, unless a majority of properties are shadowed (which is not the case here)